### PR TITLE
(fix) Fee override gets erased at hummingbot start

### PR DESCRIPTION
**Pull request by MementoRC**
**Before submitting this PR, please make sure**:

- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:
When conf_fee_overrides_TEMPLATE.yml does not contain the same list of exchanges supported by hummingbot, load_yml_into_cm mechanic does not update the ConfigVar with the values from the user's conf_fee_overrides.yml and the save ends-up erasing the value set by the user in his file
The quick fix is to populate TEMPLATE with the correct list. The rest of the files are to add test methods

**Tests performed by the developer**:
Added tests to the suite. Verified that the conf_fee_overrides.yml no longer gets erased with default values but keep the modified values

**Tips for QA testing**:

### CODE REVIEW

**Comment by carlitogetaladajr** [comment]
PRP Snapshot link:
https://snapshot.org/#/hbot-prp.eth/proposal/0xd7f87c4cfbe00a804a1152fdf3699ca5a363bd58ff87ea2b18fa631a5bf13198

File: hummingbot/templates/conf_fee_overrides_TEMPLATE.yml [hunk 1]
```diff
@@ -20,3 +20,225 @@ binance_percent_fee_token:  # BNB
 binance_maker_percent_fee:  # 0.75
 binance_taker_percent_fee:  # 0.75
 binance_buy_percent_fee_deducted_from_returns:  # True
+
+# List of supported Exchanges for which the user's conf/conf_fee_override.yml
+# will work. This file currently needs to be in sync with hummingbot list of
+# supported exchanges
+altmarkets_buy_percent_fee_deducted_from_returns:
```
Comment (aarmoa) [inline hummingbot/templates/conf_fee_overrides_TEMPLATE.yml:27]: Please remember to update the `template_version` when changing a `*TEMPLATE.yml` file

File: test/conf/conf_fee_overrides.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+binance_percent_fee_token: BNB
+binance_maker_percent_fee: 0.75
+binance_taker_percent_fee: 0.75
+binance_buy_percent_fee_deducted_from_returns: true
+
+kucoin_percent_fee_token: KCS
+kucoin_maker_percent_fee: 0.01
+kucoin_taker_percent_fee: 0.01
+kucoin_buy_percent_fee_deducted_from_returns:
```
Comment (aarmoa) [inline test/conf/conf_fee_overrides.yml:16]: The `conf_fee_overrides.yml` should not be added to the repository (no personal configuration file should be added).

File: test/templates/conf_fee_overrides_TEMPLATE.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+# Example of the fields that can be specified to override the `TradeFeeFactory` default settings.
+# If the field is missing or the value is left blank, the default value will be used.
+# The percentage values are specified as 0.1 for 0.1%.
+#
+# [exchange name]_percent_fee_token:
+# [exchange name]_maker_percent_fee:
+# [exchange name]_taker_percent_fee:
+# [exchange name]_buy_percent_fee_deducted_from_returns:  # if False, the buy fee is added to the order costs
+# [exchange name]_maker_fixed_fees:  # a list of lists of token-fee pairs (e.g. [["ETH", 1]])
+# [exchange name]_taker_fixed_fees:  # a list of lists of token-fee pairs (e.g. [["ETH", 1]])
```
Comment (aarmoa) [inline test/templates/conf_fee_overrides_TEMPLATE.yml:17]: Is it really necessary to create this file in the test folder? I would suggest creating the required test file in the test, inside a temporary directory that we make sure we delete when the test finish running.

**Review by aarmoa — CHANGES_REQUESTED** [review]
-- Reviewed in representation of the Hummingbot Foundation Technical DAO

**Comment by MementoRC** [comment]
Created new PR: https://github.com/hummingbot/hummingbot/pull/5384 in an attempt to address reviewer's concerns

**Pull request closed without merging**